tests/test_statevec_backend.py

```diff
@@ -25,13 +28,17 @@ def test_remove_one_qubit(self):
 
         np.testing.assert_almost_equal(np.abs(sv.psi.flatten().dot(sv2.psi.flatten().conj())), 1)
 
+    # TODO This is a weird test!
     def test_measurement_into_each_XYZ_basis(self):
         n = 3
         k = 0
         # for measurement into |-> returns [[0, 0], ..., [0, 0]] (whose norm is zero)
-        for state in [States.plus, States.zero, States.one, States.iplus, States.iminus]:
-            m_op = np.outer(state, state.T.conjugate())
+        # NOTE weird choice (MINUS is orthogonal to PLUS so zero)
+        for state in [BasicStates.PLUS, BasicStates.ZERO, BasicStates.ONE, BasicStates.PLUS_I, BasicStates.MINUS_I]:
+            m_op = np.outer(state.get_statevector(), state.get_statevector().T.conjugate())
+            # print(m_op)
             sv = Statevec(nqubit=n)
+            # print(sv)
             sv.evolve(m_op, [k])
             sv.remove_qubit(k)
 
```

Comment:
are we removing this?

Reply:
We'll think of modifying it later, let's keep it this way for now.